lib/app/datetime_parser.ex

```diff
@@ -0,0 +1,196 @@
+# Full credit of this module goes to https://dev.to/onpointvn/build-your-own-date-time-parser-in-elixir-50be
+# Do check the gist -> https://gist.github.com/bluzky/62a20cdb57b17f47c67261c10aa3da8b
+defmodule App.DateTimeParser do
```

Comment:
@LuchoTurtle one of the reasons I requested that you leave regular comments on the issues you're working on is precisely to know in advance when/why we _need_ to include a Date Parsing library within the MVP. 
This is **_library_ code** not **business logic**. 💭 

The actual **_edit_ `timer`** functionality should be a comparatively _tiny_ amount of code. 
Which Date/Time Parsing libraries did you look at _before_ deciding to write this File? ❓ 

We've used **`Timex`** in the past: 
https://hexdocs.pm/timex/basic-usage.html#formatting-datetimes

Which function did you need that you didn't find in one of the existing libraries? 💭

Reply:
[I did leave a comment regarding my decision of using a library code.](https://github.com/dwyl/mvp/pull/196#issuecomment-1297095672). Although I wasn't sure where I should put it in the code at first (I agree, it's not business logic, it should go to an `utils` folder or some sort), I decided to not use `Timex` because I didn't think it was worth it to import a whole library just for the sole purpose of parsing a date on a specified format. I tried to keep things lightweight on this one.

But if you think using `Timex` makes everything cleaner, I can change, no prob 😄